### ISSUE TITLE
Feature Request - Mass Email Users

### ISSUE THREAD

**Issue by Muckoma** [sct/overseerr#1554]
#### Description

Is your feature request related to a problem? If so, please provide a clear and concise description of the problem. E.g., "I'm always frustrated when [...]."
It would be extremely beneficial to have a way to bulk email users (IE, Newsletters, downtime nonfictions, other communications regarding PMS)

#### Desired Behavior

A tab in the settings to bulk email and draft an email along with a preview option. It would also be great to have the ability to select the users.

#### Additional Context
This is similar to feature within Ombi which I use occasionally to email users about expected downtime and upgrades. It would be great to see you guys incorporate it!

**Comment by jakobstavenau** [comment]
Came here to request this!

**Comment by operinko** [comment]
As far as newsletters go, something similar to Tautulli's "Recently Added to [Server]" weekly email would be great.
Sure, I can rely on Tautulli for those, but I'd much rather only have one source of emails to my users instead of.. Well, multiple.

**Comment by Muckoma** [comment]
> As far as newsletters go, something similar to Tautulli's "Recently Added to [Server]" weekly email would be great.
> Sure, I can rely on Tautulli for those, but I'd much rather only have one source of emails to my users instead of.. Well, multiple.

Yep, this is also another feature that I use now!

**Comment by douglasparker** [comment]
.

**Comment by Muckoma** [comment]
Bump

**Comment by Muckoma** [comment]
update

**Comment by CJ7SHADOW** [comment]
Bump

**Comment by HoustonDad** [comment]
Bumping this, still a great feature that allows me to fully switch from Ombi

**Comment by Muckoma** [comment]
Bump

**Comment by ohsnapword** [comment]
I'd like something like this as well.  Additionally, the ability to send users a welcome email.

**Comment by CJ7SHADOW** [comment]
Bump

**Comment by bruvv** [comment]
Not stale

**Comment by douglasparker** [comment]
The current stalebot configuration is extremely annoying...

**Comment by excessnet** [comment]
Adding an option to send a newsletter each (week?) for newly added content would be very nice!

**Comment by CJ7SHADOW** [comment]
bump

**Comment by C8opmBM** [comment]
bump! very handy for admins!

**Comment by Retalak** [comment]
Bump! This would be very helpful.

**Comment by Jaleach93** [comment]
Not Stale!

**Comment by HoustonDad** [comment]
Not stale

**Comment by Jaleach93** [comment]
Not stale, could have really used this feature tonight.

**Comment by Jaleach93** [comment]
Not stale

**Comment by bruvv** [comment]
Still waiting

**Comment by C8opmBM** [comment]
I moved to jellyseerr, since dev is way more active there.

**Comment by Jaleach93** [comment]
> I moved to jellyseerr, since dev is way more active there.

Good call! I actually forgot that I had switched over to jellyseer, so I guess this being resolved for me wouldn’t actually help me here

**Comment by bruvv** [comment]
Hu is this really completed ?
